packages/@webex/plugin-cc/src/AgentProfileService/types.ts

```diff
@@ -0,0 +1,262 @@
+/**
+ * Represents the request to UserRequest.
+ *
+ * @public
+ */
+
+export interface UserRequest {
+  /**
+   * Org id of the User
+   */
+  orgId: string;
+
+  /**
+   * CI user id of the User
+   */
+  ciUserId: string;
+}
+
+/**
+ * Represents the response from UserResponse.
+ *
+ * @public
+ */
+
+export interface UserResponse {
+  body: {
+    /**
+     * Identifier for a Desktop Profile.
+     */
+    agentProfileId: string;
+
+    /**
+     * This Specify the teams id which got assigned to the user.
+     */
+    teamIds: string[];
+
+    /**
+     * Identifier for an user profile which a Contact Center administrator has configured.
+     */
+    userProfileId: string;
```

Comment:
This should be an interface.

Reply:
This is a Profile id of a User as a string that we receive from desktopProfile API.